athlinks-ontology.ttl

```diff
@@ -366,11 +374,63 @@ dc:date rdf:type owl:AnnotationProperty .
             rdfs:comment "Exercise by swimming"@en ;
             rdfs:label "Swim Action"@en .
 
-
-###  http://schema.org/GenderType
-schema:GenderType dc:creator "nextdude"^^xsd:string ;
-                  dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime .
-
+###  http://adp.athlinks.com/schema/CohortRankingType
+:CohortRankingType rdf:type owl:Class ;
+  rdfs:subClassOf schema:Intangible ;
+  rdfs:comment "A predefined rule for associating entries to a cohort."@en ;
+  rdfs:label "Cohort Ranking Type"@en .
+
+:cohortRankingType a owl:ObjectProperty ;
+  rdfs:domain :CohortRanking ;
+  rdfs:range [
+    rdf:type owl:Class ;
+    owl:unionOf(
+      :CohortRankingType
+      schema:string
```

Comment:
We should use xsd:string; schema:string doesn't exist in the api.

Reply:
Fixing